packages/redux/src/exchanges/__tests__/reducer.test.ts

```diff
@@ -148,6 +148,34 @@ describe('exchanges reducer', () => {
     });
   });
 
+  describe('result() reducer', () => {
+    it('should return the initial state', () => {
+      const state = reducer(undefined, randomAction).result;
+
+      expect(state).toBe(initialState.result);
+    });
+
+    it.each([
+      actionTypes.CREATE_EXCHANGE_SUCCESS,
+      actionTypes.FETCH_EXCHANGE_SUCCESS,
+    ])('should handle %s action type', actionType => {
+      const expectedResult = mockState.exchanges.result;
+
+      expect(
+        reducer(undefined, {
```

Comment:
Do not use `undefined` as the state here, please. Use a state that has isLoading: true at least.

Reply:
Fixed 🙂